singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -745,9 +829,35 @@ class PTESolverRhoT
   PORTABLE_INLINE_FUNCTION
   Real Init() {
     InitBase();
+
+    // Decide which material's volume fraction should be set by the
+    // sum of the others. If an index is passed in through mix params,
+    // we use that. Otherwise we do our best.
+    //
+    // We probably want to exclude the material with the largest
+    // volume fraction, but we don't know which one that will be. We
+    // guestimate it as the material with the largest mass fraction,
+    // though this may not always be correct.
+    //
+    if (params_.pte_reduced_system_exclude_idx >= 0) {
+      ms = params_.pte_reduced_system_exclude_idx;
+    } else {
+      ms = 0;
+      Real xmax = 0;
+      Real alphamax = 0;
+      for (std::size_t m = 0; m < nmat; ++m) {
+        Real x = robust::ratio(rhobar[m], rho_total);
+        if (x > xmax) {
+          xmax = x;
+          ms = m;
+        }
+      }
+    }
+    PORTABLE_REQUIRE(ms < nmat, "Index of material whose volume fraction will depend on "
+                                "the others must be less than nmat");
+
     Residual();
-    // Leave this in for now, but comment out because I'm not sure it's a good idea
-    // TryIdealPTE(this);
```

Comment:
`TryIdealPTE` is still in the code, but I don't think this comment is still adding value, so I removed it. We've put a lot more thought into our initial guesses since this was attempted.

Reply:
Is there a reason for keeping `TryIdealPTE` at this point?

Reply:
Yeah I'm not sure. I'd be fine removing it

Reply:
Doesn't need to be a part of this MR, but I figured I'd bring it up